web/frontend/src/components/camera/cameras-list.vue

```diff
@@ -0,0 +1,105 @@
+<script setup lang="ts">
+import type {
+  commonApi,
+  Client,
+} from '@viamrobotics/sdk';
+
+import Camera from './camera.vue';
+import PCD from '../pcd/pcd.vue';
+
+interface Props {
+  resources: commonApi.ResourceName.AsObject[],
+  client: Client,
+  parentName: string
+}
+const props = defineProps<Props>();
+
+const openCameras = $ref<Record<string, boolean | undefined>>({});
+const refreshFrequency = $ref<Record<string, string | undefined>>({});
+
+const triggerRefresh = $ref(false);
+const selectedMap = {
```

Comment:
Looks like we are duplicating this in `./camera.vue`, maybe we move this to `lib/camera-state.ts` and share it between the two components.